packages/common/src/services/audius-backend/AudiusBackend.ts

```diff
@@ -1800,16 +1804,109 @@ export const audiusBackend = ({
       const tx = await getCreateAssociatedTokenAccountTransaction({
         feePayerKey: SolanaUtils.newPublicKeyNullable(phantomWallet),
         solanaWalletKey: SolanaUtils.newPublicKeyNullable(address),
-        mintKey: audiusLibs.solanaWeb3Manager.mints.audio,
-        solanaTokenProgramKey: audiusLibs.solanaWeb3Manager.solanaTokenKey,
-        connection: audiusLibs.solanaWeb3Manager.getConnection()
+        mintKey: new PublicKey(env.WAUDIO_MINT_ADDRESS),
+        solanaTokenProgramKey: new PublicKey(TOKEN_PROGRAM_ID),
+        connection
+      })
+      const { signature, lastValidBlockHeight, recentBlockhash } =
+        await window.solana.signAndSendTransaction(tx)
```

Comment:
This feels kind of wild. I can see it's how we were doing it before. But it feels like it's just going to straight crash if the user doesn't have a solana-compatible extension installed that makes window.solana available?

Reply:
i agree it is kinda wild. connecting wallets is currently broken for some other downstream reason as well tho (phantom at least) so gonna call this not part of this PR if that's ok?

Reply:
> connecting wallets is currently broken for some other downstream reason as well tho (phantom at least)

weird, [our analytics](https://app.amplitude.com/analytics/audius/dashboard/qoi8jv9) didn't seem to represent that?

Do we have a bug for this/what errors are you seeing?